Update metadata.txt to include a changelog for 1.1.0

havatv opened the pull request:
Update metadata.txt to include changelog for 1.1.0

mkondratyev85 commented:
Current version of raster tracer is 0.1.1. I believe next release could be v.0.2, not 1.1.0 as in your proposal.

havatv commented on metadata.txt:
```diff
@@ -20,7 +20,10 @@ repository=https://github.com/mkondratyev85/raster_tracer/
 hasProcessingProvider=no
 # Uncomment the following line and add your changelog:
-# changelog=
+changelog=1.1.0
```
```suggestion
changelog=0.2.0
```

havatv commented:
> Current version of raster tracer is 0.1.1. I believe next release could be v.0.2, not 1.1.0 as in your proposal.

Sorry about that. I only added the changelog, and for some reason I used 1.1.0 (probably a cut and paste from another plugin changelog)
I have now changed *version* and updated *changelog* in `metadata.txt`.

mkondratyev85 commented:
I've added new tag v0.2 and uploaded zip to QGIS plugins repository https://plugins.qgis.org/plugins/raster_tracer/ and now it waits for approval.
